packages/ui/package.json

```diff
@@ -42,6 +42,7 @@
     "@fortawesome/react-fontawesome": "0.2.0",
     "@sovryn/tailwindcss-config": "*",
     "classnames": "2.3.1",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
As I mentioned earlier, I don't know if we need it but if we decide that we do, pls remove `^`.